ext/semian/sysv_semaphores.h

```diff
@@ -97,4 +106,17 @@ get_semaphore(int key);
 void *
 acquire_semaphore_without_gvl(void *p);
 
+#ifdef DEBUG
```

Comment:
I kind of want to keep this definition after removing the debug prints, as its very useful for debugging.